releases.go

```diff
@@ -251,36 +251,45 @@ func releasesCreate(db *gorm.DB, release *Release) (*Release, error) {
 	return release, nil
 }
 
-func newServiceApp(release *Release) *scheduler.App {
+func newSchedulerApp(release *Release) *scheduler.App {
+
 	var processes []*scheduler.Process
 
 	for name, p := range release.Formation {
-		processes = append(processes, newServiceProcess(release, name, p))
+		processes = append(processes, newSchedulerProcess(release, name, p))
```

Comment:
Ah yeah. It was named `Service` because package `scheduler` used to be named package `service`, and this just never got changed.